graphrag/index/__main__.py

```diff
@@ -79,20 +129,20 @@
     args = parser.parse_args()
 
     if args.resume and args.update_index:
-        msg = "Cannot resume and update a run at the same time."
```

Comment:
Interesting. Previously Ruff failed when there wasn't a dot. Guess it now changed :)